res/TensorFlowLiteRecipes/Net_Shape_Add_000/test.recipe

```diff
@@ -0,0 +1,46 @@
+# test for constant folding of Shape operation
+
+operand {
+  name: "ifm"
+  type: INT32
+  shape { dim: 4 }
+}
+operand {
+  name: "constant"
+  type: FLOAT32
+  shape { dim: 1 dim: 32 dim: 64 dim: 32 }
```

Comment:
if not needed with this size, can we use smaller vales like `[1, 2, 3, 4]`?

Reply:
Ok, large size is not essential. Updated :)